frontend/pages/my-data.vue

```diff
@@ -0,0 +1,204 @@
+<script setup lang="ts">
+const prefix = 'pages/my-data'
+
+const { fromQueryReactive } = useURLParams()
+const { t } = useI18n()
+const pactaClient = usePACTA()
+const { loading: { withLoading } } = useModal()
+
+const tt = (s: string) => t(`${prefix}.${s}`)
+
+const selectedPortfolioIdsQP = fromQueryReactive('pids')
+const selectedPortfolioGroupIdsQP = fromQueryReactive('pgids')
+const tabQP = fromQueryReactive('tab')
```

Comment:
You could update `fromQueryReactive` to take a second, optional parameter that sets a fallback default, would clean up the `(<thing>.value ?? <default>)`s below.

Reply:
Great idea. Done.

Reply:
This is a huge win - it also allows us to do the implicit clever thing of not setting QPs when they're the defaults, simplifying URLs while not making them any less precise. Thanks for this suggestion!